js/Game.js

```diff
@@ -26,14 +23,18 @@ class Game {
     this.changeTurn();
   }
 
-  generateFivePuzzles() {
-    //Chooses 5 random puzzles
-    //iterate over all of the banks and make them into one large array
-    //generate a random number between zero and the length of the indexes 
-    // conditional if we run into the same index to generate a new one
-    //then generate 5 new puzzle classes based on that
-    //pass into puzzle constructor 
+  generateWheel() {
+    const wheelOne = new Wheel();
+    return [wheelOne]
+  }
 
+  generateFivePuzzles() {
```

Comment:
Although I can see needing to create a new instance of `Round` if you have a class of `Round` that keeps track of round-related things - I don't know that I would create all those instances at once. It would make more sense to have a new instance of `Round` created when a new round starts... maybe in a `nextRound` or `startNewRound` method that Game creates.